ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2407,7 +2407,33 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-upi
+- as: baremetalds-assisted-hypershift-agent-mce-dualstack-mgmt-f7
+  cluster: build05
+  cron: 14 4 7,14,21,28 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4v6
+        NETWORK_TYPE=OVNKubernetes
+        VM_EXTRADISKS=true
+        VM_EXTRADISKS_LIST=vda
+        VM_EXTRADISKS_SIZE=1024G
+        NUM_EXTRA_WORKERS=3
+        EXTRA_WORKER_VCPU=8
+        EXTRA_WORKER_MEMORY=16384
+        EXTRA_WORKER_DISK=100
+      IP_STACK: v4v6
```

Comment:
Why do we need the same info `IP_STACK` twice ?

Reply:
They are used for different steps, https://github.com/openshift/release/blob/45a1fafbbd842daff9b791650bf91d2d887125bd/ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__periodics.yaml#L141

Reply:
It indicates something we can improve in the design in the future.

Not a blocker to this PR though.